recipes/gpustat/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "gpustat" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2f43421dbcd9ebd8caf179aeb5f78ac123786033fb9a4310ce0f8e18b25eb03e
+
+build:
+  skip: True  # [win]
```

Comment:
Could this be a noarch package? [See here](http://conda-forge.org/docs/maintainer/knowledge_base.html#noarch%20builds)

It’s okay if windows then fails due to missing dependencies as the feedstock will only be built in Linux.